org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/XtextGeneratorTemplates.xtend

```diff
@@ -466,7 +466,7 @@ class XtextGeneratorTemplates {
 			
 				@Override
 				protected �'org.osgi.framework.Bundle'.typeRef� getBundle() {
-					return �'org.eclipse.core.runtime.Platform'.typeRef�.getBundle(�eclipsePluginActivator�.PLUGIN_ID);
+					return �'org.osgi.framework.FrameworkUtil'.typeRef�.getBundle(�eclipsePluginActivator�.class);
```

Comment:
is this on classpath? can we use real typeref?

Reply:
yes
<img width="1293" alt="Bildschirmfoto 2020-10-26 um 08 51 50" src="https://user-images.githubusercontent.com/265597/97146660-81ffe180-1768-11eb-9968-00dd07d15f5d.png">

Reply:
I think the point was to use the real class in the template.

Reply:
yes

Reply:
org.osgi.framework.FrameworkUtil.typeRef with no ''